sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -510,7 +463,10 @@ FormRecognizerAsyncClient formRecognizerAsyncClient = new FormRecognizerClientBu
     .buildAsyncClient();
 ```
 
-* Recognize receipts : [RecognizeReceiptsAsync][recognize_receipts_async]
+* Recognize business card from a URL: [RecognizeBusinessCardFromUrlAsync][recognize_business_cards_from_url_async]
```

Comment:
same here... business cards and identity are URL, invoice is by file, and receipt has both.
For consistency, consider all having both